lib/joystick.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const {
+  isConnected,
```

Comment:
With more and more methods like this, I think we should put a prefix before methods like `isConnected`, `update`, etc.

e.g.

- `joystickIsConnected`
- `joystickUpdate`

Just like what `touch` did.